test/integration/simple_example/controls/forseti.rb

```diff
@@ -184,5 +185,38 @@
       end
     end
   end
+end
+
+control 'forseti-org-iam' do
+  title "Validate organization roles of SA"
+  describe command("gcloud organizations get-iam-policy #{org_id} --filter='bindings.members:#{forseti_server_service_account}' --flatten='bindings[].members' --format='csv(bindings.role)'") do
```

Comment:
In most/all of our tests that shell out to `gcloud` we generally use the `json` format. Is there a particular reason why we chose CSV for this test?

Reply:
It was easier for me to list the roles in CSV and parse in ruby. https://github.com/marko7460/terraform-google-forseti/blob/feature/add_missing_role/test/integration/simple_example/controls/forseti.rb#L196-L201.
I can fix the code to deal with json if that is important for consistency with other projects and tests.

Reply:
@marko7460 yeah, let's just switch this to JSON. We don't benefit from using CSV and it adds complexity and cognitive load; let's go with the common convention.